test/system/800-config.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Test specific configuration options and overrides
+#
+
+load helpers
+
+@test "podman CONTAINERS_CONF - CONTAINERS_CONF in conmon {
+    skip_if_remote "can't check conmon environment over remote"
+
+    # Make an innocuous containers.conf in a non-standard location
+    export CONTAINERS_CONF="$PODMAN_TMPDIR/containers.conf"
+    cat >$CONTAINERS_CONF <<EOF
+[engine]
+runtime="$OCI_RUNTIME"
```

Comment:
There is no guarantee that `$OCI_RUNTIME` will be defined.

Reply:
Good point.  I've changed it to be `${OCI_RUNTIME:-crun}` instead.